### PR TITLE
 Rewrite callbacks for avoid using redirect_to :back

### PR DESCRIPTION
**Pull request by ymasiuk**

### CODE REVIEW

File: app/controllers/admin/blockchains_controller.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ def create
       if @blockchain.save
         redirect_to admin_blockchains_path
       else
-        flash[:alert] = @blockchain.errors.full_messages
+        flash[:alert] = @blockchain.errors.full_messages.first
```
Comment (ysv) [inline app/controllers/admin/blockchains_controller.rb:24]: Why did you change to `.first` ?

**Comment by ysv** [comment]
@ymasiuk Create issue please
Also once approved open same pr to master

File: app/controllers/admin/blockchains_controller.rb (hunk 1 above)
Comment (ymasiuk) [reply app/controllers/admin/blockchains_controller.rb:24]: @ysv without `.first` we will see error like `["ERROR_TEXT"]` with `.first` it would be just like string without any characters.

**Comment by shal** [comment]
@ymasiuk Please rebase, your patch doesn't work now.

**Comment by ymasiuk** [comment]
@shal done

**Review by ysv — APPROVED** [review]
